main.go

```diff
@@ -67,21 +68,32 @@ func main() {
 func doCheckExistenceAndParse() {
 	switch {
 	case strings.Contains(path, "Gopkg.lock"):
-		dep := packages.Dep{}
-		dep.GopkgPath = path
-		if dep.CheckExistenceOfManifest() {
-			dep.ProjectList, _ = parse.GopkgLock(path)
-			var purls = processPackages(dep)
-			var packageCount = len(purls)
+		slices := strings.Split(path, "/Gopkg.lock")
 
-			checkOSSIndex(purls, packageCount)
+		workingDir := slices[0]
+		if(workingDir == "Gopkg.lock"){
+			workingDir, _ = os.Getwd()
+		}
+		getenv := os.Getenv("GOPATH")
+		ctx := dep.Ctx{
+			WorkingDir: workingDir,
+			GOPATH:     getenv,
+			GOPATHs:    []string{getenv},
```

Comment:
Is the duplicate GOPATH necessary?

https://godoc.org/github.com/golang/dep#Ctx

Reply:
Nope. Looks like I can get rid of GOPATH. I started with that one and it failed with this error. I then added GOPATHs with success but didn't realize GOPATH wasn't needed at all. So I'll clean it up. 

```
=== RUN   TestExtractPurlsFromManifestUsingDep
Gopkg.lock
Gopkg.toml
--- FAIL: TestExtractPurlsFromManifestUsingDep (0.01s)
    dep_int_test.go:39: /Users/zender/dev/projects/oss/nancy/packages/dir047508631/src/projectname is not within a known GOPATH/src
FAIL```

Reply:
Hmm... I wonder if it depends on where you run `nancy`

```
    GOPATH         string        // Selected Go path, containing WorkingDir.
    GOPATHs        []string      // Other Go paths.
```

Reply:
Not sure.....here is what i just tried

Install a build that only sets GOPATH and does not set GOPATHs. Fails with an error.
![image](https://user-images.githubusercontent.com/1099906/64500773-8deec280-d28c-11e9-9a63-bba31b45661d.png)


Use that same build but try to run it from a different directory. Same error
![image](https://user-images.githubusercontent.com/1099906/64500785-9d6e0b80-d28c-11e9-993e-de912d7a5dd4.png)



Install a new build that only sets GOPATHs and does not set GOPATH. Works with no issues from either location. 
![image](https://user-images.githubusercontent.com/1099906/64500797-afe84500-d28c-11e9-8193-90fa46bb745c.png)
![image](https://user-images.githubusercontent.com/1099906/64500803-baa2da00-d28c-11e9-95da-c9583089d9b8.png)